### PR TITLE
Feature/rating

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added styling and function to rate on rating component.

### CODE REVIEW

**Review by Ankur9669 — APPROVED** [review]
I have only reviewed rating bar, which looks fine to me

**Review by bossmonkey98 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.scss
+color.css
+*.map
```
Comment (Ranjeetk489) [inline .gitignore]: what is this for?

File: css/components/avatar/index.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  width: 3rem;
+  height: 3rem;
+}
+.avatar-badge-container {
+  position: relative;
+  /* transform:scale(1);
+    border-radius: 50%;
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
```
Comment (Ranjeetk489) [inline css/components/avatar/index.css]: remove commented out code

File: css/components/avatar/index.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+}
+.avatar-badge {
+  right: 0%;
+  top: 0%;
+  position: absolute;
+  border-radius: 50%;
+  min-width: 2rem;
+  padding: 0.25rem;
```
Comment (Ranjeetk489) [inline css/components/avatar/index.css]: use px

File: css/components/badge/index.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+.badge {
+  right: -5%;
+  top: -5%;
+  position: absolute;
+  display: grid;
+  place-content: center;
+  border-radius: 50%;
+  width: 1.5rem;
+  padding: 0.25rem;
```
Comment (Ranjeetk489) [inline css/components/badge/index.css]: use px

File: css/components/button/index.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
+button,
+input[type="submit"] {
+  cursor: pointer;
+  outline: none;
+  border: none;
+  padding: 0.5rem 0.6rem;
```
Comment (Ranjeetk489) [inline css/components/button/index.css]: use px

File: css/components/image/index.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  object-fit: cover;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.img-rounded {
+  border-radius: 50%;
+}
+
+.img-responsive {
```
Comment (Ranjeetk489) [inline css/components/image/index.css]: delete it

File: css/components/input/index.css [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+    height: .7rem;
+    border: solid white;
+    border-width: 0 .2rem .2rem 0;
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+  }
+
+
+  .radio-container {
```
Comment (Ranjeetk489) [inline css/components/input/index.css]: remove it

File: css/components/modal/index.css [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    filter:blur(0px);
+
+    transform: scale(1);
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+
+ 
+
+}
+
+  /* .slide-down{
```
Comment (Ranjeetk489) [inline css/components/modal/index.css:42]: remove it

**Review by Ranjeetk489 — APPROVED** [review]
After making the changes your code will be good to be merged and if possible try going through your code again.

**Pull request merged**